moveit_planners/chomp/chomp_motion_planner/src/chomp_parameters.cpp

```diff
@@ -80,4 +80,18 @@ void ChompParameters::setRecoveryParams(double learning_rate, double ridge_facto
   this->planning_time_limit_ = planning_time_limit;
   this->max_iterations_ = max_iterations;
 }
+
+const std::vector<std::string> ChompParameters::VALID_INITIALIZATION_METHODS{ "quintic-spline", "linear", "cubic",
+                                                                              "fillTrajectory" };
+
+bool ChompParameters::setTrajectoryInitializationMethod(std::string method)
+{
+  if (std::find(VALID_INITIALIZATION_METHODS.cbegin(), VALID_INITIALIZATION_METHODS.cend(), method) !=
+      VALID_INITIALIZATION_METHODS.end())
```

Comment:
```suggestion
  if (VALID_INITIALIZATION_METHODS.count(method) != 1)
```
could be used if you switch from `std::vector<std::string>` to `std::set<std::string>` (or `.contains(method)` starting with c++ 20 ...)

Reply:
We (sadly) don't have C++20, or we could improve so much code with Ranges... :smile: 

I don't see why I should use a rb-tree for a list of 4 elements, so I will keep this as is.